### PR TITLE
[Mono] Fix llvm-jit mode

### PR DESCRIPTION
**Pull request by fanyang-mono**
Forgot to turn on LLVM for llvm-jit mode.

cc: @EgorBo

### CODE REVIEW

File: build/mono-scenarios.yml [hunk 1]
```diff
@@ -36,9 +36,9 @@ parameters:
     - displayName: "Core CLR"
       arguments: --property runtime=coreclr
     - displayName: "Mono - JIT"
-      arguments: --application.useMonoRuntime jit --property runtime=mono-jit
+      arguments: --application.useMonoRuntime jit --property runtime=mono-jit --application.environmentVariables MONO_ENV_OPTIONS="--llvm"
```
Comment (sebastienros) [inline build/mono-scenarios.yml:39]: Is that to enable LLVM?

Comment (fanyang-mono) [reply build/mono-scenarios.yml:39]: oops, wrong line. Let me fix it.

**Comment by EgorBo** [comment]
🎉

**Comment by EgorBo** [comment]
image

**Comment by sebastienros** [comment]
Was the regression expected, looks like from 6 to 7, even if the ENV needed to be present before that.

**Comment by EgorBo** [comment]
> Was the regression expected, looks like from 6 to 7, even if the ENV needed to be present before that.

well it wasn't really expected but it's identified, a change to coreclr/libraries broke vectorization for non-LLVM Mono (and even for Mono-LLVM for some parts) - https://github.com/dotnet/runtime/issues/61484

**Comment by EgorBo** [comment]
@sebastienros any idea why there are no mono runs since 2/8/2022?

**Comment by sebastienros** [comment]
All LLVM benchmarks are failing. There is a generic error message. @fanyang-mono can you try to run one of the LLVM ones and see what the errors are?

**Comment by EgorBo** [comment]
> All LLVM benchmarks are failing. There is a generic error message. @fanyang-mono can you try to run one of the LLVM ones and see what the errors are?

```
crank  --profile aspnet-citrine-lin --config https://raw.githubusercontent.com/aspnet/Benchmarks/main/scenarios/platform.benchmarks.yml  --scenario plaintext --application.options.displayOutput true --application.useMonoRuntime llvm-jit --property runtime=mono-llvm --application.environmentVariables MONO_ENV_OPTIONS="--llvm"
```
I see no errors

**Comment by sebastienros** [comment]
You are missing `--framework 7.0 --application.options.collectCounters true` to reproduce what is actually running on the CI. However it also works for me. Any AOT one though will fail with an error that Fan should look into (dedicated mono support in the agent).
I should have more logs tomorrow morning about why this llvm-jit one is not showing up though it's working for us (it's Monday and the logs from last week got deleted last night).

**Pull request merged**